src/server/utils/devCdnFactory.js

```diff
@@ -166,22 +167,15 @@ export const oneAppDevCdnFactory = ({
         remoteModuleBaseUrls
       );
       const remoteModuleBaseUrlOrigin = new URL(knownRemoteModuleBaseUrl).origin;
-      try {
-        const remoteModuleResponse = await fetch(`${remoteModuleBaseUrlOrigin}/${incomingRequestPath}`, {
-
-          headers: { connection: 'keep-alive' },
-          agent: new ProxyAgent(),
-        });
-        reply
-          .code(remoteModuleResponse.statusCode)
-          .type(path.extname(req.url))
-          .send(remoteModuleResponse.body);
-      } catch (err) {
-        const status = err.statusCode === 'ERR_NON_2XX_3XX_RESPONSE' ? err.response.statusCode : 500;
-        return reply
-          .code(status)
-          .send(err);
-      }
+      const remoteModuleResponse = await fetch(`${remoteModuleBaseUrlOrigin}/${incomingRequestPath}`, {
```

Comment:
probably still need the try / catch to handle network errors (Promise rejection)?